projects/helm/chartmuseum/Makefile

```diff
@@ -0,0 +1,35 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=chartmuseum
+REPO_OWNER=helm
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base
+SOURCE_PATTERNS=./cmd/chartmuseum
+
+BINARY_TARGET_FILES=chartm
+
+HAS_S3_ARTIFACTS=true
+IMAGE_NAMES=
+
+PRECOMPILE_PREPARE=harbor_patch
```

Comment:
Same comment as above, I think you can do this by adding a pre-req to the binary target